db/db_tunables.h

```diff
@@ -1285,4 +1285,13 @@ REGISTER_TUNABLE("poll_in_pgfree_recover", "Poll pgfree recovery handler.",
                  TUNABLE_BOOLEAN, &gbl_poll_in_pg_free_recover,
                  EXPERIMENTAL | INTERNAL, NULL, NULL, NULL, NULL);
 
+REGISTER_TUNABLE("portmux_port", "Specifies portmux port", TUNABLE_INTEGER,
```

Comment:
there already is a portmux_port (remove?):
https://github.com/mponomar/comdb2/blob/b0560811f6811fa9e9e10fcec3cd5cdff3834818/db/db_tunables.h#L937
There also is a portmux_bind_path which may be similar to the "portmux_sock" you added next.

Reply:
we use the two i mentioned above in the testsuite (see setup), if the user defines a PMUXPORT.

Reply:
Thanks Adi, I didn't think these existed.  No reason to have 2 sets of options for the same thing.